core/src/utils/internal/scrollbars.ts

```diff
@@ -20,11 +20,17 @@ const internalRemoveScrollbars = () => {
 
 let internalRevert = noop;
 
+/**
+ * A function to remove the scrollbars on the body element. It can be reverted using the `revertScrollbars` function.
+ */
 export const removeScrollbars = () => {
 	internalRevert();
 	internalRevert = internalRemoveScrollbars();
 };
 
+/**
+ * A function to revert the removal of scrollbars performed by the `removeScrollbars` function.
```

Comment:
`A function to revert the removal of scrollbars performed by the {@link removeScrollbars} function.`